src/v/config/configuration.cc

```diff
@@ -945,6 +944,13 @@ configuration::configuration()
       "How often to trigger background compaction.",
       {.needs_restart = needs_restart::no, .visibility = visibility::user},
       10s)
+  , tombstone_retention_ms(
+      *this,
+      "tombstone_retention_ms",
+      "The retention time for tombstone records in a compacted topic.",
```

Comment:
Could we document the tiered storage limitations here?

Reply:
Added